src/types/transactions.ts

```diff
@@ -70,3 +70,35 @@ export enum TRANSACTION_METHODS {
   DeployTokenAuthority = 'deployTokenAuthority',
   Approve = 'approve',
 }
+
+export enum BatchKeys {
```

Comment:
The batch keys are actually method names, so they should be merged with this: https://github.com/JoinColony/colonyCDapp/blob/185cae2e8073c2241e5425a2a50bd9886e6d68de/src/types/transactions.ts#L67C19-L72